### DISCUSSION TITLE
Issue with shadcn-ui component installation path in WXT project

### DISCUSSION THREAD

**Discussion by only-issues** [wxt-dev/wxt#1192] [Q&A]
I'm encountering an issue with shadcn-ui component installation in my WXT project. The components are being installed outside of my project directory, despite having the correct path configuration in `components.json`.

### Current Setup
My `components.json` configuration:
```json
{
  "$schema": "https://ui.shadcn.com/schema.json",
  "style": "new-york",
  "tailwind": {
    "config": "tailwind.config.js",
    "css": "./src/assets/style.css",
    "baseColor": "zinc",
    "cssVariables": false
  },
  "rsc": false,
  "tsx": true,
  "aliases": {
    "utils": "@/lib/utils",
    "components": "@/components"
  }
}
```

### The Problem
When I run the command `pnpm dlx shadcn@latest add button`, the component gets installed outside my project directory:

Current structure after installation:
```
/Projects/My Project/components.json
/Projects/My Project/package.json
/Projects/My Project/src/entrypoints/
/Projects/My Project/src/components/
/Projects/My Project/src/.../
/Projects/src/components/ui/button.tsx  <- Wrong location!
```

### Root Cause Analysis
I believe this is happening because WXT creates its own `tsconfig.json` in `.wxt/tsconfig.json` with a path configuration where `"@/"` is set to `"../src/*"`. It seems that shadcn-ui's installation process doesn't properly handle this nested tsconfig situation, leading to incorrect path resolution.

### Attempted Solutions
I've tried adding aliases in `wxt.config.ts`:
```typescript
// Attempt 1
shadcnSrc: resolve('src')

// Attempt 2
shadcnSrc: './src'
```

However, both attempts result in WXT adding `..` prefix in `.wxt/tsconfig.json`, which doesn't solve the issue.

And finally, I want to know if there is any other way to fix it?

**Comment by aklinker1** [comment]
You could define your own paths in the root TS config, and not inherit any from the one inside the .wxt directory. 

If the aliases change you would have to update your own list, which is a little annoying. But maybe that will work? 

Of course, you should also open an issue in shadcn, seems like they're not parsing TS config files correctly. 

Maybe also search nuxt for for a similar issue. They use the same structure of the ../prefix in the .nuxt directory.
